[docs] GCP MySQL IAM authentication support (#38943)
gravitational/teleport: docs/pages/includes/database-access/cloudsql_service_credentials.mdx

> The Teleport Database Service must have credentials of `teleport-db-service`
> GCP service account we created in step 3 in order to be able to generate IAM auth
> tokens.

Reviewer: ```suggestion
The Teleport Database Service must have credentials for the `teleport-db-service`
GCP service account we created in step 3 in order to generate IAM auth
tokens.
```

Reviewer: or possibly even just drop the details about why it needs credentials exactly, e.g.:

```suggestion
The Teleport Database Service must have credentials for the `teleport-db-service`
GCP service account we created in step 3.
```

Author: nice, i like less text.

If the Teleport Database Service is hosted on a VM, you can [change the
attached service
account](https://cloud.google.com/compute/docs/instances/change-service-account).

Alternatively, set the `GOOGLE_APPLICATION_CREDENTIALS` environment variable to
point to the JSON credentials file you downloaded earlier. If you are using
`systemd` to start `teleport`, then you should edit the service's
`EnvironmentFile` to include the following env var:
```code
$ echo 'GOOGLE_APPLICATION_CREDENTIALS=/path/to/credentials.json' | sudo tee -a /etc/default/teleport
```

See [Authenticating as a service
account](https://cloud.google.com/docs/authentication/production) in the Google

> Cloud documentation for more details.

Reviewer 2: I would specify the additional details a user would expect to find on this page so they can get through this guide as quickly as possible. Otherwise, our style guide advises against linking to external guides.

Author: moved to next steps
